code/modules/atmospherics/machinery/components/unary_devices/cryo.dm

```diff
@@ -83,59 +83,61 @@
 /obj/machinery/atmospherics/components/unary/cryo_cell/update_icon()
 	cut_overlays()
 
+	if(panel_open)
+		add_overlay("pod-panel")
+
 	if(state_open)
 		icon_state = "pod-open"
-	else if(occupant)
-		var/image/occupant_overlay
+		return
 
-		if(ismonkey(occupant)) // Monkey
-			occupant_overlay = mutable_appearance(CRYOMOBS, "monkey")
+	if(occupant)
+		var/image/occupant_overlay
 
-		else if(isalienadult(occupant))
+		if(ishuman(occupant) || islarva(occupant) || (isanimal(occupant) && !ismegafauna(occupant))) // Mobs that are smaller than cryotube

```

Comment:
instead of checking for megafauna, it should probably check for the mob size([the defines are here](https://github.com/tgstation/tgstation/blob/357903f1f80b71f3c69386f45318022eafbba050/code/__DEFINES/mobs.dm#L27))

Reply:
I pretend to work on this later and make the cryo crop the sprites, but for now I'm leaving it as is.